### DISCUSSION TITLE
Anyone who succeeded finetuning pixart-xl-1024ms on custom dataset?

### DISCUSSION THREAD

**Discussion by youngwanLEE** [PixArt-alpha/PixArt-alpha#107] [Q&A]
I'm finetuning the pixart-xl-1024ms model on a custom dataset, but it's not going well, and I'm wondering if I should continue trying. I'm looking for motivation to keep going, so I'm curious if there are any success stories out there.

**Comment by ruizhaocv** [comment]
Yes, I finetuned the 1024ms model on my customized dataset, and the training process is quite smooth. You can have a try. Be careful to set the config correctly, especially the lewei_scale: https://github.com/orgs/PixArt-alpha/discussions/104#discussioncomment-8897912.
